README.md

```diff
@@ -134,14 +134,14 @@ response instead.
 1. Create a `main.py` file with the following contents:
 
    ```python
-   def hello(request):
-        return "Hello world!"
+   def hello(data, context):
+        print("Received", context.event_id)
```

Comment:
```suggestion
        print(f"Subscription ID: {context.resource['name']}")
        print(f"Message body: {base64.b64decode(event['data']).decode('utf-8')}")
        print(f"Publish timestamp: {context.timestamp}")
        return "OK", 200
```

Show how to extract important fields of a Pub/Sub message from `event` and `context`.

Reply:
I'm not sure this is the right place to do that, are there Pub/Sub docs we could link to instead?

Also, background functions do not have return values or status codes.

Reply:
We can use this link: https://cloud.google.com/functions/docs/tutorials/pubsub#functions_helloworld_pubsub_tutorial-python